scripts/bb-test-cleanup.sh

```diff
@@ -177,6 +177,9 @@ export CODE_COVERAGE_BRANCH_COVERAGE=${TEST_CODE_COVERAGE_BRANCH_COVERAGE:-0}
 set -x
 cd "${ZFS_BUILD}"
 
+# Unload modules
+sudo -E $ZFS_SH -vu
```

Comment:
Is this necessary anymore? The PERF builder doesn't run cleanup anymore.

Reply:
Right, I'll drop it.